docs/source/reducing_memory_usage.md

```diff
@@ -129,6 +129,42 @@ training_args = SFTConfig(..., padding_free=True, model_init_kwargs={"attn_imple
 </hfoption>
 </hfoptions>
 
+## Activation offloading
+
+Activation offloading is a memory efficiency technique that reduces GPU VRAM usage by temporarily moving activation tensors to CPU RAM during the forward pass and bringing them back only when needed for the backward pass. This significantly reduces peak memory usage at the cost of slightly increased training time.
+
+To enable activation offloading in your SFT training configuration:
+
+</hfoption>
```

Comment:
Out of curiosity, what do these tags do?

Reply:
it shows alternatives for different trainers: https://github.com/huggingface/doc-builder?tab=readme-ov-file#options